pom.xml

```diff
@@ -22,14 +22,16 @@
         <version.smallrye-context-propagation>1.0.13</version.smallrye-context-propagation>
         <version.smallrye-converters>1.0.0</version.smallrye-converters>
         <version.smallrye-fault-tolerance>4.3.0</version.smallrye-fault-tolerance>
-        <version.smallrye-graph-ql>1.0.5</version.smallrye-graph-ql>
+        <version.smallrye-graphql>1.0.5</version.smallrye-graphql>
         <version.smallrye-health>2.2.3</version.smallrye-health>
         <version.smallrye-jwt>2.2.0</version.smallrye-jwt>
         <version.smallrye-metrics>2.4.2</version.smallrye-metrics>
-        <version.smallrye-openapi>1.2.4</version.smallrye-openapi>
```

Comment:
I believe that we should keep OpenAPI in the 1.x range since it targets MP OpenAPI 1.1. SR OpenAPI 2.x targets OpenAPI 2.0 that is coming on MP 4.0, but the BOM targets MP 3.3.

Reply:
SmallRye OpenAPI 2.x is still against MP 1.x. But it contains the Spring Web support.

Reply:
So not an issue :)